src/main/scala/firrtl/Emitter.scala

```diff
@@ -1078,6 +1092,16 @@ class VerilogEmitter extends SeqTransform with Emitter {
         emit(Seq("`endif // SYNTHESIS"))
       }
 
+      if (formals.keys.nonEmpty) {
+        emit(Seq("`ifdef FORMAL"))
+        for ((clk, content) <- formals if content.nonEmpty) {
+          emit(Seq(tab, "always @(posedge ", clk, ") begin"))
+          for (line <- content) emit(Seq(tab, tab, line))
```

Comment:
This is SystemVerilog, so there should be some way to prevent emitting this for the "Verilog" flow.

Reply:
Or just to print a warning? Which tool would actually reject these constructs because of the `.v` file ending?

Reply:
It's not about the file extension -- if we have a "Verilog" backend, it should produce Verilog (95, 2001, or 2005).

Reply:
OK. So the solution would be to move the code that Tom added to the `VerilogEmitter` class at the end of the `Emitter.scala` file and remove the `StageUtils.dramaticWarning`?
What do you suggest?

Reply:
I've fixed this in the latest changes, please take a look!

Reply:
Thanks, will do!